src/ts/QuerySelectors.ts

```diff
@@ -21,6 +21,7 @@ export enum QuerySelectors {
     // Story
     storyImage = '.y-yJ5',
     storyCloseButton = '.K_10X',
```

Comment:
Perhaps more like this.
```typescript
storyCloseButton = '.K_10X, ._g3zU'
```

That way you can remove all the logic from the StoryDowloader and the querySelector will pick the appropriate element